libs/service-portal/information/src/components/ChildView/ChildView.tsx

```diff
@@ -194,59 +244,112 @@ const ChildView: FC<Props> = ({
         <Box marginY={3} />
         {(person?.parent1 || person?.parent2 || loading) && (
           <>
-            <Parents
+            <TwoColumnUserInfoLine
               title={formatMessage({
-                id: 'sp.family:custody-and-parents',
-                defaultMessage: 'Forsjá & foreldrar',
-              })}
-              label={formatMessage({
                 id: 'sp.family:parents',
                 defaultMessage: 'Foreldrar',
               })}
-              parent1={person?.nameParent1}
-              parent2={person?.nameParent2}
+              label={formatMessage(m.name)}
+              firstValue={person?.nameParent1}
+              secondValue={person?.nameParent2}
               loading={loading}
             />
-            <Parents
+            <Divider />
+            <TwoColumnUserInfoLine
               label={formatMessage(m.natreg)}
-              parent1={person?.parent1 ? formatNationalId(person.parent1) : ''}
-              parent2={person?.parent2 ? formatNationalId(person.parent2) : ''}
+              firstValue={
+                person?.parent1 ? formatNationalId(person.parent1) : ''
+              }
+              secondValue={
+                person?.parent2 ? formatNationalId(person.parent2) : ''
```

Comment:
Wondering :) -> Can there be more than 2 parents that share custody?

Reply:
Not yet